### DISCUSSION TITLE
Bug

### DISCUSSION THREAD

**Discussion by VaBanck** [sle118/squeezelite-esp32#292] [General]
![image](https://github.com/sle118/squeezelite-esp32/assets/75133860/9b9f3889-6284-4c4c-bae3-7fc61b19a143)

It works for a few minutes and reboots.

**Comment by sle118** [comment]
You will definitely need to provide more details, as it is impossible to understand what is going on or the context of the issue

**Comment by philippe44** [comment]
You probably don't have an LMS server and use squeezelite-esp32 in an usual way. Please see https://github.com/sle118/squeezelite-esp32#additional-configuration-notes-from-the-web-ui

**Reply by VaBanck** [reply]
It doesn't happen on other versions, "-s -disable" then works
